repository/src/main/java/org/apache/atlas/util/lexoRank/system/LexoNumeralSystem64.java

```diff
@@ -0,0 +1,41 @@
+package org.apache.atlas.util.lexoRank.system;
```

Comment:
I feel we should keep this number system, as if in future we need to move to a bigger system, basically needing more ranks, this could come in handy.
Not just for GCT but for any other entity-types.